src/runner.py

```diff
@@ -118,13 +129,24 @@ def create(
         logger.info("Creating runner: %s", self.config.name)
 
         try:
+            if self.config.issue_metrics:
+                try:
+                    self._shared_fs = shared_fs.create(self.config.name)
+                except errors.CreateSharedFilesystemError:
+                    logger.exception(
+                        "Unable to create shared filesystem for runner %s. "
+                        "Will not create metrics for this runner.",
+                        self.config.name,
+                    )
+
             self.instance = self._create_instance(image, resources)
             self._start_instance()
             # Wait some initial time for the instance to boot up
             time.sleep(60)
```

Comment:
Do we need this sleep considering there's a _wait_boot_up function below it - I see in the definition of it a retry decorator, so I assume it's doing the same thing?

Reply:
This was not introduced by this PR, but I believe the sleep should prevent the logging of an error message (https://github.com/canonical/github-runner-operator/blob/589e139e843cce84b778bf1b513ed9a16c7b5ee6/src/utilities.py#L80) , which happens on every retry.

Reply:
Sounds good, I also noticed it is not part of the PR, wanted to ask. Thanks!